crates/pixi_config/src/lib.rs

```diff
@@ -888,6 +896,26 @@ pub fn config_path_global() -> Vec<PathBuf> {
     .collect()
 }
 
+impl<'c> From<&'c Config> for rattler_repodata_gateway::ChannelConfig {
+    fn from(config: &'c Config) -> Self {
+        let default_source_config = config
+            .repodata_config
+            .as_ref()
+            .map(|config| SourceConfig {
+                jlap_enabled: !config.disable_jlap.unwrap_or(false),
+                zstd_enabled: !config.disable_zstd.unwrap_or(false),
+                bz2_enabled: !config.disable_bzip2.unwrap_or(false),
+                cache_action: Default::default(),
+            })
+            .unwrap_or_default();
+
+        Self {
+            default: default_source_config,
+            per_channel: Default::default(),
+        }
+    }
+}
```

Comment:
Maybe move this back to pixi and use a function instead? I think this is the only reason we need the `gateway` feature of repodata_gateway or even the crate at all.

Reply:
Sure, done.